blog/2015/11/python-lesson-rewrite.html

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog
+root: ../../..
+author: Matt Davis
+title: "Python Lesson Rewrite"
+date: 2015-11-14
+time: "11:30:00"
+category: ["Lessons", "Python"]
+---
+<!-- start excerpt -->
+<section>
+  <p>
+    Recent
+    <a href="http://software-carpentry.org/blog/2015/09/thinking-about-teaching.html">
+      analysis
+    </a>
+    and
+    <a href="http://software-carpentry.org/blog/2015/10/python-debriefing-summary.html">
+      introspection
+    </a>
+    regarding Software Carpentry's Python lesson has led us to conclude that
+    the lesson would benefit from a complete rewrite.
+    The subcommittee on lesson development
+    <a href="http://pad.software-carpentry.org/python-lesson-planning-2015-10-14">met twice</a>
+    to plan much about the new lessons (yes, that's plural).
+    This post will describe our plans for the new lessons.
+  </p>
+  <p>
+    We're going to write two new Python lessons, one intended for students who
+    are completely new to programming (the "novice" lesson),
+    and another for students with some programming experience (the "intermediate" lesson).
```

Comment:
It is important to me to specify that this is for people with some programming experience *in a language other than python*